lib/result.js

```diff
@@ -1,3 +1,32 @@
-module.exports = () => {
-  console.log('wiby result')
+const test = require('./test')
+const github = require('./github')
+
+module.exports = async function (pkg) {
+  const branch = await getBranchName()
+  const [org, repo] = await test.getGitHubOrgRepo(pkg).catch(err => { throw (err) })
+  const resp = await github.getChecks(org, repo, branch).catch(err => { throw (err) })
```

Comment:
```suggestion
  const [org, repo] = await test.getGitHubOrgRepo(pkg)
  const resp = await github.getChecks(org, repo, branch)
```
a catch that just re-throws is redundant, and will also break node's ability to show the line number from where it originated.

Reply:
Fixed